src/model-viewer-base.ts

```diff
@@ -332,30 +333,44 @@ export default class ModelViewerElementBase extends UpdatingElement {
   }
 
   /** @export */
-  async toBlob(mimeType?: string, qualityArgument?: number): Promise<Blob> {
-    return new Promise(async (resolve, reject) => {
-      if ((this[$canvas] as any).msToBlob) {
-        // NOTE: msToBlob only returns image/png
-        // so ensure mimeType is not specified (defaults to image/png)
-        // or is image/png, otherwise fallback to using toDataURL on IE.
-        if (!mimeType || mimeType === 'image/png') {
-          return resolve((this[$canvas] as any).msToBlob());
-        }
-      }
-
-      if (!this[$canvas].toBlob) {
-        return resolve(await dataUrlToBlob(
-            this[$canvas].toDataURL(mimeType, qualityArgument)));
-      }
-
-      this[$canvas].toBlob((blob) => {
-        if (!blob) {
-          return reject(new Error('Unable to retrieve canvas blob'));
-        }
-
-        resolve(blob);
-      }, mimeType, qualityArgument);
-    });
+  async toBlob(options: {
```

Comment:
Let's make `options` optional (`options?`)

Reply:
I tried to do that, except `const {mimeType, qualityArgument, idealAspect} = options!;` says can't destructure undefined. I was hoping it would just destructure them to undefined as well, but apparently not. Is there a clean way to resolve that?

Reply:
Done.